src/modules/3-display.js

```diff
@@ -0,0 +1,68 @@
+export default function displayTasks() {
+  const taskList = document.querySelector('.task-list');
+  let parsedArr = JSON.parse(localStorage.getItem('taskArr')) || [];
+  taskList.innerHTML = '';
+  parsedArr.map((task) => {
+    const taskItem = document.createElement('li');
+    taskItem.innerHTML = `<form class="task-form b-bottom box">
+      <input name="completed" type="checkbox" class="checkbox">
+      <textarea name="description" rows="1" class="task-text full">${task.description}</textarea>
+      <button type="button" class="delete btn">
+      <img src="https://cdn-icons-png.flaticon.com/512/542/542724.png" alt="update-icon">
+      </button>
+      <button type="submit" class="update btn">
+      <img src="https://cdn-icons-png.flaticon.com/512/1828/1828743.png" alt="update-icon">
+      </button>
+      </form>`;
+    taskList.appendChild(taskItem);
+    const deleteBtn = taskItem.querySelector('.delete');
+    const updateBtn = taskItem.querySelector('.update');
+    const updateText = taskItem.querySelector('.task-text');
```

Comment:
you should implement these variables of the begging of the project for future changes